src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 
 ### Preferences Framework ###
 
+#### Panel Changes ####
+Changes and additions are being made to UI Options in order to improve its compliance with accesibility standards.
+
+##### Message Bundle Keys and Selectors #####
+
+######Additions######
+* `textSizeDescr`
+* `textFontDescr`
+* `lineSpaceDescr`
+* `contrastDescr`
+
+######Changes######
+<table>
+<tr><th> 1.5</th><th>2.0</th></tr>
+<tr>
+    <td><code>speakChoiceLabel</code></td>
+    <td><code>speakDescr</code></td>
+</tr>
+<tr>
+    <td><code>tocChoiceLabel</code></td>
+    <td><code>tocDescr</code></td>
+</tr>
+<tr>
+    <td><code>simplifyChoiceLabel</code></td>
+    <td><code>simplifyDescr</code></td>
+</tr>
+</table>
+
+##### CSS Selectors #####
```

Comment:
@acheetham what do you think about this section being called "CSS Selectors". I'm a bit worried that it may be confused with CSS styling rather than Javascript DOM selection.